src/views/Bridge/BridgePanel.tsx

```diff
@@ -790,31 +596,28 @@ export const BridgePanel: React.FC = () => {
                 size="lg"
                 width="100%"
                 isDisabled={
-                  !fromAccount ||
+                  !from ||
                   isLoadingBalance ||
-                  !targetAccount ||
-                  (!collectible && (!amount || balance?.lt(amount))) ||
+                  !to ||
+                  (!collectible && !amount) ||
                   isTransferring ||
-                  isInvalidTargetAccount ||
                   targetAccountNeedDepositStorage
                 }
                 isLoading={isTransferring}
                 spinner={
                   <PulseLoader color="rgba(255, 255, 255, .9)" size={12} />
                 }
-                onClick={isReverse ? onBurn : onRedeem}
+                onClick={isNearToAppchain ? onBurn : onRedeem}
               >
-                {!fromAccount
+                {!from
                   ? "Connect Wallet"
-                  : !targetAccount
+                  : !to
                   ? "Input Target Account"
-                  : isInvalidTargetAccount || targetAccountNeedDepositStorage
+                  : targetAccountNeedDepositStorage
                   ? "Invalid Target Account"
                   : !collectible
                   ? !amount
                     ? "Input Amount"
-                    : balance?.lt(amount)
-                    ? "Insufficient Balance"
```

Comment:
why remove the "Insufficient Balance" tip?

Reply:
> why remove the "Insufficient Balance" tip?

It's not tip, it's button text. This validation will be shown when user submit.

Reply:
The old code make the button too complex. Now put all validation to the submit step.